mantaray/backend.py

```diff
@@ -502,7 +502,21 @@ def send_privmsg(self, nick_or_channel: str, text: str) -> None:
             done_event=SentPrivmsg(nick_or_channel, text),
         )
 
+
+    def kick(self, channel: str, nick: str, reason: str | None = None) -> None:
+        if reason is None:
+            reason=''
+            
+        print(channel, nick, reason)
+        self._send_soon(
+            "KICK",
+            channel,
+            nick,
+            " Reason:" + reason
```

Comment:
That's not how it works. It was better before the latest commit. I meant that you should write code like this:

```
if reason is None:
    send KICK without reason
else:
    send KICK including reason
```